src/AppLovinMAXAdView.js

```diff
@@ -175,6 +196,11 @@ AdView.propTypes = {
    * A boolean value representing whether or not to enable adaptive banners. Note that adaptive banners are enabled by default as of v2.3.0.
    */
   adaptiveBannerEnabled: PropTypes.bool,
+
+  /**
+   * A boolean value representing whether or not to enable auto refresh. Note that auto refresh is enabled by default.
```

Comment:
Native SDK docs reference it as "auto-refresh":

```suggestion
   * A boolean value representing whether or not to enable auto refresh. Note that auto-refresh is enabled by default.
```